src/main/java/resources/PersonsResource.java

```diff
@@ -81,8 +82,18 @@ public Notes getNote(@PathParam("personId") Long personId,
     @Path("/updateNote")
     @Consumes(MediaType.APPLICATION_JSON)
     @UnitOfWork
-    public Notes createOrUpdate(Notes note){
-        return notesDao.createOrUpdate(note);
+    public Response createOrUpdate(Notes note){
+        Date now = new Date();
+        Calendar c = Calendar.getInstance();
+        c.setTime(note.getDate()); // Now use today date.
```

Comment:
This comment suggests that it uses today's date - so adding 5 days might result in moving to next week